doc/development/api/general.rst

```diff
@@ -60,7 +60,7 @@ Dashboards
    :members: event_dashboard_widgets, user_dashboard_widgets
 
 
-Displaying of log entries
+Display of log entries
 -------------------------
```

Comment:
Dunno how tolerant the .rst parser is, but maybe w should adjust the number of dashes in the line below?